__tests__/app.test.ts

```diff
@@ -107,3 +108,24 @@ describe("/api/users/:_id", () => {
     expect(user.error).toBe("400 Error: Invalid ID Syntax");
   });
 });
+
+describe.only("/api/articles", () => {
+  test("should return array of all articles", async () => {
+    const req = {} as NextRequest;
+    const res = (await getAllArticles(req)) as NextResponse;
+
+    const data = await res.json();
+
+    expect(res.status).toBe(200);
+    expect(Array.isArray(data.articles)).toBe(true);
+
+    data.articles.forEach((article: any) => {
```

Comment:
If we extract the types into a type file we can replace this with the article type